drafts/str4d-stratum/draft1.rst

```diff
@@ -0,0 +1,256 @@
+Abstract
+========
+
+This ZIP describes the Zcash variant of the Stratum protocol, used by miners to
+communicate with mining pool servers.
+
+
+Specification
+=============
+
```

Comment:
Also specify connection management and the roles of the parties, e.g. that the miner is a JSON-RPC client, and it makes a TCP connection (on what port?) to a Zcash-Stratum server which is a JSON-RPC server.